src/main/java/Pages/Services/ServicesPage.java

```diff
@@ -1,9 +1,9 @@
-package Web.SiteTree.Services;
+package Pages.Services;
```

Comment:
all classes that describe pages, elements, helpers should be located in folders under src/main/java
remove them from src/test/java and keep only test files there

Reply:
Fixed.